addons/website/views/snippets/s_mega_menu_odoo_menu.xml

```diff
@@ -5,10 +5,10 @@
     <section class="s_mega_menu_odoo_menu pt16 o_colored_level o_cc o_cc1">
         <div class="container">
             <div class="row">
-                <div class="col-md-6 col-lg pt16 pb24">
+                <div class="col pt16 pb24">
```

Comment:
We already talked about that before in the team: IMO I think this should either be a global behavior or not a behavior at all. Not a fan of making the col resize different depending on where you are using it :grimacing: Let's remove that while waiting for a decision. You can create a task about that elsewhere and suggest it later.

Reply:
Hello, this was done to avoid breaking the mega_menu inside its offcanvas, since we can have the mega_menu displayed in a >lg using the hamburger navigation template it can yield a terrible result. The goal of this col was to provide a consistent result on the mega_menu between offcanvas (large viewport), offcanvas mobile, and normal navigation
![image](https://github.com/odoo/odoo/assets/118886338/f5126611-39de-4b1c-92e9-30b3fad54af0)